### ISSUE TITLE
Arrow Keys Keyboard Navigation experience just like the default Windows Explorere

### ISSUE THREAD

**Issue by wanderseeme** [files-community/Files#2510]
**Is your feature request related to a problem? Please describe.**

```
Keyboar navigation with arrow keys are currently not supported well, re-produce steps:

1. visit any folder
2. press UP or DOWN arrow key to navigate up or down, but not working
3. you can now select a folder/file to make arrow keys work
4.but when you press Alt+UP to go one level up, the arrow keys can't work again!
```

**Describe the solution you'd like**

`Capture arrow keys wisely, always make them navigate through files pane!`

**Describe alternatives you've considered**

`This is a "simple" feature from a user's perspective, no alternatives.`

**Additional context**
Add any other context or screenshots about the feature request here. Ex: What device type/UI layout and app version?

```
I think the steps above is already self-explaining enought.
I just wish you the best here, thanks for developing such an awesome UWP File app!
```

**Comment by Thomas1664** [comment]
Might me a duplicate of #2477

**Comment by yaichenbaum** [comment]
@wanderseeme This issue should be resolved with #2636.
